.github/workflows/haskell-ci.yml

```diff
@@ -176,3 +176,42 @@ jobs:
         run: |
           rm -f cabal.project.local
           $CABAL v2-build $ARG_COMPILER --disable-tests --disable-benchmarks all
+
```

Comment:
Thanks @gdziadkiewicz, for adding CI to macOS and windows!

It is, however, not a good idea in general to modify a generated file.  Next time I run `haskell-ci`, you work will be lost.  Please (1) move this into its own `.yml` file, or (2) create a patching mechanism that will append this to the `haskell-ci` generated file.  Probably (1) is simpler, except that then an additional batch is needed.

Reply:
I implemented (1) in #205.

Reply:
I missed the fact that haskell-ci.yml is generated by `haskell-ci`. Thanks for fixing that :)